webapp/src/shared/components/Input/EditUserForm.tsx

```diff
@@ -2,39 +2,85 @@ import { useState } from 'react';
 import { Button, ButtonVariant, Input, InputVariant } from '../';
 import './EditUserForm.css';
 
-export default function EditUserForm() {
-  const [fullName, setFullName] = useState('');
-  const [email, setEmail] = useState('');
+type EditUserProps = {
+  origFullName: string;
+  origEmail: string;
+};
+
+export default function EditUserForm({
+  origFullName,
+  origEmail,
+}: EditUserProps) {
+  const [fullName, setFullName] = useState(origFullName);
+  const [email, setEmail] = useState(origEmail);
+
+  async function updateData() {
+    const requestOptions = {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        fullName,
+        email,
+      }),
+    };
+    await fetch('http://localhost:3000/api/v1/users', requestOptions);
```

Comment:
praise:
thanks for connecting it to the backend!

Reply:
Thanks